sixtron_flash/cli.py

```diff
@@ -87,14 +87,19 @@ def main(jlink_device, file_path):
     command_file.write(command_file_content)
     command_file.close()
 
+    if jlink_probe is not None:
+        probe = "-USB {}".format(jlink_probe)
+    else:
+        probe = ""
+
     # Flash target
     if os.name == "nt":
         executable = "JLink.exe"
     else:
         executable = "JLinkExe"
     # fmt: off
-    cmd = "{} -Device {} -if SWD -Speed 4000 -ExitOnError 1 -NoGui 1 -CommandFile \"{}\" ".format(
-        executable, jlink_device, command_path
+    cmd = "{} -Device {} {} -if SWD -Speed 4000 -ExitOnError 1 -NoGui 1 -CommandFile \"{}\" ".format(
+        executable, jlink_device, probe, command_path
```

Comment:
As above, I would group options by "scope":
```suggestion
    cmd = f"{probe} -CommandFile \"{command_path}\" -Device {jlink_device} {executable}"
```
And use formatted string literals: https://docs.python.org/3/tutorial/inputoutput.html#tut-f-strings